packages/react/test/specs/components/Slider/Slider-test.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { ReactWrapper } from 'enzyme'
+
+import { mountWithProvider as mount } from 'test/utils'
+import { isConformant, handlesAccessibility } from 'test/specs/commonTests'
+import Slider from 'src/components/Slider/Slider'
+
+const testValue = '30'
+const getInputDomNode = (sliderComp: ReactWrapper): HTMLInputElement =>
+  sliderComp.find('input').getDOMNode() as HTMLInputElement
+
+const setUserInputValue = (sliderComp: ReactWrapper, value: string) => {
+  sliderComp.find('input').simulate('change', { target: { value } })
+}
+
+describe('Slider', () => {
+  describe('conformance', () => {
+    isConformant(Slider, {
+      eventTargets: {
+        onChange: 'input',
+        onKeyDown: 'input',
+        onKeyPress: 'input',
+        onKeyUp: 'input',
+      },
+    })
+  })
+
+  describe('auto-controlled', () => {
+    it('sets slider value from user when the value prop is not set (non-controlled mode)', () => {
+      const sliderComp = mount(<Slider />)
+      const domNode = getInputDomNode(sliderComp)
```

Comment:
Why we need to capture this from DOM? I believe that `expect(wrapper.find('input').prop('value')).toEqual('30')` should be enough

Reply:
I copied this from `Input-test.tsx`. Any major concerns with it?

Reply:
It's a strange thing to test DOM instead of React in this case. But you can keep it, probably `Input-tests` should be also refactored...